src/css/_dropdown.scss

```diff
@@ -0,0 +1,61 @@
+@use "theme/borders";
```

Comment:
I moved the contents of this file from `_header.scss`. Nothing changed outside of where I put comments